src/docs/travelRequest.js

```diff
@@ -1,13 +1,65 @@
+/**
+* @swagger
+* /api/v1/trip/save:
+*   post:
+*     tags:
+*       - Trips
+*     name: save trip
+*     summary: create trip request
+*     consumes:
+*        - multipart/form-data
```

Comment:
You may use `- application/json`
this link can help you how to de describe Request Body: [swagger docs](https://swagger.io/docs/specification/2-0/describing-request-body/)